### ISSUE TITLE
Use of Colour in "Tips for Getting Started in Web Accessibility"

### ISSUE THREAD

**Issue by davidberman** [w3c/wai-quick-start#211]
Problem: poor colour contrast in the six items in this tile, causing lack of clarity for those with color deficit as to which is current.
Potential solution: choose a higher-contrast colour pair and/or make the underscores persistent.

**Comment by iadawn** [comment]
Do you have a preferred solution?

**Comment by davidberman** [comment]
My preferred solution would be to make the underscores persistent.

**Comment by iadawn** [comment]
Thanks for your preference, I have played around a bit and underlines seem to add quite a bit of visual noise to the links. I think that this simply makes highlighting the current page more challenging. I wonder if using an indicator (as in the primary nav where chevrons are used) would be a more suitable option?

**Comment by iadawn** [comment]
I have introduced a couple of changes to try to make the current page within the resource more identifiable within the section navigation.

* Slight change to background color of navigation to increase overall contrast
* Inclusion of a current page indicator that matches primary navigation indicator

Interested in your thoughts on this approach.

**Comment by James-Green** [comment]
I like the changes.

**Comment by davidberman** [comment]
I see your indicator,
and it does help a bit,
however I'd prefer the underline... or perhaps the words for the current item would be bold with the others regular? That would also work, and also would follow the spirit of WCAG.

- David


On 2015-09-09 08:03, Kevin White wrote:
>
> Thanks for your preference, I have played around a bit and underlines seem to add quite a bit of visual noise to the links. I think that this simply makes highlighting the current page more challenging. I wonder if using an indicator (as in the primary nav where chevrons are used) 
> would be a more suitable option?
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/w3c/wai-quick-start/issues/211#issuecomment-138888390>.
>

-- 
David Berman, RGD, FGDC LinkedIn <http://www.linkedin.com/in/bermandavid> Twitter @davidberman <http://www.twitter.com/davidberman> Facebook <http://www.facebook.com/davidbberman> Skype davidberman.com <skype:davidberman.com?chat> Google Plus 
<https://plus.google.com/u/0/+DavidBermanCom/posts>
David Berman Communications | berman@davidberman.com | @davidberman | blog <http://www.designedgecanada.com/author/david-berman>
+1-613-728-6777 | 340 Selby Avenue, Ottawa K2A 3X6

High Level Advisor, United Nations | GDC ethics chair | Ico-D Sustainability chair | Carleton University Access Network chair
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
/Accessibility courses:/ Ottawa | Europe | Vancouver | Victoria
/Upcoming:/ Toronto | Mexico City | Dublin | Belfast | Korea | Bahrain
Watch David on CBS <http://www.wtoc.com/story/17588481/scad-plans-revitalization> | Do Good book news: <http://www.dogoodbook.com/> "Don't just do good design ... do good!"

This message may contain proprietary information. Unauthorized disclosure/copying/distribution of contents prohibited.


---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus

**Comment by iadawn** [comment]
@davidberman, I have changed it so that only the current item is in bold. I am resisting using underlines a bit as I feel that it does have quite a significant impact on the legibility of this navigation and would create an exception to eht WAI navigation styling. This is one of a couple of indicators of location of the resource, the others being the primary navigation and the title.

The other thing that might be worth considering is using a background on the current item:

<img width="163" alt="background_on_current" src="https://cloud.githubusercontent.com/assets/2444840/9809098/218877ba-585d-11e5-9142-0dae07407bb9.png">

I am a little bit hesitant as this does add much more visual salience to this area, and I know that this causes problems for some people.

**Comment by davidberman** [comment]
I think the bolding is working.

I also like the background on the current item, and I don't think the visual salience is a problem: actually I think it does an excellent job of helping identify the purpose of the entire object.
My only concern with it is how you are then showing focus when the user is in the midst of selecting one of the other pages. Can you show a screen capture of that?


On 2015-09-11 03:15, Kevin White wrote:
>
> @davidberman <https://github.com/davidberman>, I have changed it so that only the current item is in bold. I am resisting using underlines a bit as I feel that it does have quite a significant impact on the legibility of this navigation and would create an exception to eht WAI 
> navigation styling. This is one of a couple of indicators of location of the resource, the others being the primary navigation and the title.
>
> The other thing that might be worth considering is using a background on the current item:
>
> background_on_current <https://cloud.githubusercontent.com/assets/2444840/9809098/218877ba-585d-11e5-9142-0dae07407bb9.png>
>
> I am a little bit hesitant as this does add much more visual salience to this area, and I know that this causes problems for some people.
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/w3c/wai-quick-start/issues/211#issuecomment-139470252>.
>



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus

**Comment by iadawn** [comment]
If possible, I would like to keep the visual noise in this section to a minimum. With the primary navigation, the change to the bolding, and the visual cue from the chevron is enough?

**Comment by yatil** [comment]
I like it and it works for me.

**Comment by davidberman** [comment]
The chevron is well implemented, but I don't think it is as effective as the lighter-coloured background you mocked up above. I do think you have solved the problem, so I find the solution as is acceptable.
